.github/workflows/swift-toolchain.yml

```diff
@@ -2668,7 +2691,7 @@ jobs:
   snapshot:
     runs-on: ubuntu-latest
     needs: [context, smoke_test]
-
+    if: github.event_name != 'pull_request'
```

Comment:
Even these could be upstreamed

Reply:
Will do!

Reply:
https://github.com/compnerd/swift-build/pull/742